clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -51,6 +53,10 @@ const (
 	latencyWindowSize = 5 * time.Minute
 )
 
+var (
+	enableViolations = pflag.Bool("enable-violations-api-responsiveness-prometheus", false, "Whether to enable violations in the API call responsivness measurment done via Prometheus")
```

Comment:
Open question, are we going to have a flag per each prometheus measurement? 

Have you considered moving this to the measurment parameters, e.g. something like
```
  - Identifier: APIResponsivenessPrometheus
    Method: APIResponsivenessPrometheus
    Params:
      action: start
      failTestOnViolation: true|false
```

This could be implemented in a generic way for each prometheus based measurment, wdyt?

Reply:
Good questions. Assuming we want to gate somehow all major changes to clusterloader2, I'd say yes. 

I like your idea or having this in the configuration file. However I think more thought needs to be put into that (e.g. an open question is whether it will support well gating (overrides?)) and I'd rather have it in following PR. 

Let's submit this one and start to enable the violations from this measurement and in the meanwhile let's figure out something more elegant and scalable in terms of num of measurements.

Reply:
Removing a flag is a painful, multi-step process.  I'd prefer to avoid it, especially given that I think we're not yet in a place where we can enable these violations, right? 
We'll first need to get a few runs of this reported into perf-dash to see and understand what are the differences. So I guess we should focus on implementing the perf-dash integration first. 

You can also run the tests manually with the violations enabled if you want to check if/which (resource, verb, scope) triples would fail.

Reply:
1. This is non-production code, so removing flags is not a big deal (two PR at most). 
2. Differences are mostly due to current measurement being less strict (further away from SLO definition). But I agree it make sense to integrate perf-dash first. I assumed this would work out-of-the-box, but it turned out it's not the case. 
3. I'll close this PR and run those tests manually. I'll follow up with another PR that would try to solve this problem in a generic way as you've proposed.